src/s1/components/banner/bannerStacks.jsx

```diff
@@ -174,6 +182,7 @@ export default class BannerStacks extends Component {
 						maxHeight: maxHeightModal,
 					}}
 				>
+					{ shouldShowCloseX && <div class={style.closeX} onClick={this.handleAcceptAll}>&times;</div>}
```

Comment:
can you change this so we can track when user clicks "Close" vs "acceptAll" will be good to see telemetry on that.

```
onClick={this.handleClose}
//  ....

// further up 
handleClose = () => {
    this.handleAcceptAll('acceptAllClose');
}

handleAcceptAll = (clickCategoty = 'acceptAll') => {
 // change logger click category 
};

```

Reply:
Added these to the 3 banners.